README.md

```diff
@@ -0,0 +1,75 @@
+[![Build Status](https://travis-ci.org/kuzzleio/sdk-java.svg?branch=master)](https://travis-ci.org/kuzzleio/sdk-java) [![codecov.io](http://codecov.io/github/kuzzleio/sdk-java/coverage.svg?branch=master)](http://codecov.io/github/kuzzleio/sdk-java?branch=master)
```

Comment:
You can remove codecov shield. We do not have unit tests on SDK Java. 
Or put the Golang one :)